cmake/std/atdm/vortex

```diff
@@ -0,0 +1 @@
+ats2
```

Comment:
I would just remove this file.  The only new directories we need are:

```
cmake/std/atdm/ats2/
cmake/ctest/drivers/atdm/ats2/
```

and subdirs under those.

Reply:
I think I need to take another look at how the scripts will find ats2 -- but these symlinks do the job.

Reply:
With setting `hostnameMatchSystemName=ats2`, you can delete this symlink.